aspnet/web-pages/content-guide.md

```diff
@@ -45,7 +45,7 @@ WebMatrix is a set of tools that provides an easy way to build Web sites for Mic
 
 **Read**
 
-- [Microsoft Web Camps training kit for WebMatrix](http://trainingkit.webcamps.ms/WebMatrix.htm)
+
 - [Submit a bug or suggestion](https://go.microsoft.com/fwlink/?LinkId=195940)
```

Comment:
You can delete this entire line--it's targeted at the UserVoice forum, which no longer exists.